core/record/_record/recordReader.js

```diff
@@ -60,11 +60,15 @@ export const getNodeChildren = (node) => (record) =>
     R.map((uuid) => getNodeByUuid(uuid)(record))
   )(record)
 
-export const getNodeChildrenByDefUuid = (parentNode, nodeDefUuid) => (record) =>
+export const getNodeChildrenByDefUuidUnsorted = (parentNode, nodeDefUuid) => (record) =>
```

Comment:
Maybe we can use some selectors with memoization, this library can be used without redux state

https://github.com/reduxjs/reselect#creating-a-memoized-selector